src/util/openfaas/index.js

```diff
@@ -92,8 +92,18 @@ const isFunctionDeployed = (functionName) => {
 
 const setFunctionInCache = (functionName) => {
   const funcList = functionListCache.get(FUNC_LIST_KEY) || [];
```

Comment:
since we are maintaining unique function names in the array, should we  just do funcList.includes(functionName)

Reply:
Absolutely. added